templates/errors/404.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'main/base.html' %}
+{% block title %} 404 - Page not found! {% endblock %}
+{% block beta_error %}{% endblock %}
+{% block content %}
+    <div class="uk-background-muted">
+        <div class="uk-section">
+            <div class="uk-text-center"><code class="uk-text-large">AttributeError: Page not located!</code></div>
+        </div>
+    </div>
+
+    <div class="uk-section uk-text-center">
+        <p>The page you requested does not exist, try go back <a href="/">home</a> and try again, if you think this is
```

Comment:
"The page you requested does not exist. Try going back __home__."
"If you think this shouldn't happen, open a ticket on our __GitHub__."

Reply:
This template is not in use anymore. This template will be removed.

Reply:
understandable